configs/animal/2d_kpt_sview_rgb_img/topdown_heatmap/animalpose/hrnet_animalpose.yml

```diff
@@ -1,40 +1,42 @@
 Collections:
 - Metadata:
     Architecture:
-    - HRNet
-  Name: hrnet_animalpose
+    - HRNet (CVPR'2019)
+  Name: animal 2D Keypoint topdown_heatmap animalpose
   Paper:
-  - http://openaccess.thecvf.com/content_CVPR_2019/html/Sun_Deep_High-Resolution_Representation_Learning_for_Human_Pose_Estimation_CVPR_2019_paper.html
-  - http://openaccess.thecvf.com/content_ICCV_2019/html/Cao_Cross-Domain_Adaptation_for_Animal_Pose_Estimation_ICCV_2019_paper.html
+  - Title: Deep high-resolution representation learning for human pose estimation
```

Comment:
Why does the collection has two papers?

Reply:
One collection is corresponding to a dataset and a method, each can involve multiple papers.